src/modules/sway/language.cpp

```diff
@@ -179,8 +179,7 @@ auto Language::init_layouts_map(const std::vector<std::string>& used_layouts) ->
 }
 
 Language::XKBContext::XKBContext() {
-  context_ = rxkb_context_new(RXKB_CONTEXT_NO_DEFAULT_INCLUDES);
-  rxkb_context_include_path_append_default(context_);
+  context_ = rxkb_context_new(RXKB_CONTEXT_LOAD_EXOTIC_RULES);
```

Comment:
BTW, I removed the `RXKB_CONTEXT_NO_DEFAULT_INCLUDES` flag (and the call to `rxkb_context_include_path_append_default`) since I found this to be quite ugly:
```
context_ = rxkb_context_new(static_cast<rxkb_context_flags>(RXKB_CONTEXT_NO_DEFAULT_INCLUDES | RXKB_CONTEXT_LOAD_EXOTIC_RULES));
```
and I see no reason to avoid loading the default rules at first, when the next statement adds them explicitly.